modules/microshift-install-rhel-image-mode-publish-image.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-publish-image_{context}"]
+= Publishing the bootc image to the remote registry
+
+Publish your bootc image to the remote registry so that the image can be used for running the container on another host, or for when you want to install a new operating system with the bootc image layer.
+
+.Prerequisites
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
```

Comment:
Strictly speaking, the 1st requirement is not necessary for publishing images. 
You need to be logged into the same host where the local image was built.

Reply:
Carried over from building.